conf/nonclangable.conf

```diff
@@ -269,6 +269,11 @@ CFLAGS:remove:pn-mozjs:toolchain-clang = "-fno-tree-vrp"
 CFLAGS:append:pn-ffmpeg:riscv64 = " -march=rv64gczbb"
 CFLAGS:append:pn-ffmpeg:riscv32 = " -march=rv32gczbb"
 
+# otherwise systemd efi fails to build
+# error: the 'sse' unit is not supported with this instruction set
+TUNE_CCARGS:remove:pn-systemd:toolchain-clang = "-mfpmath=sse"
+TUNE_CCARGS:remove:pn-systemd-boot:toolchain-clang = "-mfpmath=sse"
```

Comment:
I am not sure if its should be moved here. It improves the situation, where it reports errors while gcc does not. SSE is not available everywhere and clang rightly points it out. also look at ltp recipe